src/mixins/filing-template-mixin.ts

```diff
@@ -1139,7 +1155,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
    * @param filing the filing to parse
    */
   private storeStaffPayment (
-    filing: AlterationFilingIF | CorrectionFilingIF | ConversionFilingIF | ChgRegistrationFilingIF
+    filing: AlterationFilingIF | CorrectionFilingIF | ConversionFilingIF |
+            ChgRegistrationFilingIF | SpecialResolutionFilingIF
```

Comment:
Not needed unless you also use `buildStaffPayment()`, which you don't, but maybe you should?

Reply:
Looks good.